internal/genny/assets/webpack/webpack.go

```diff
@@ -88,15 +88,19 @@ func New(opts *Options) (*genny.Generator, error) {
 
 func installPkgs(r *genny.Runner, opts *Options) error {
 	command := "yarnpkg"
+	args := []string{"install", "--silent"}
 
 	if !opts.App.WithYarn {
 		command = "npm"
+		args = []string{"install", "--no-progress", "--save"}
 	} else {
 		if err := installYarn(r); err != nil {
 			return err
 		}
+		if err := r.Exec(exec.Command(command, []string{"set", "version", "berry"}...)); err != nil {
+			return err
+		}
 	}
-	args := []string{"install", "--no-progress", "--save"}
 
 	c := exec.Command(command, args...)
```

Comment:
Since the command generates more than 4K lines of logs for yarn, which is too much verbose, we should add `logFilters` to `.yarnrc.yml` that I mentioned in the issue comment before executing the command. I tried to find the equivalent command-line options or environment variables but could not find them.